auter

```diff
@@ -151,7 +152,12 @@ function reboot_server() {
   fi
 
   logit "INFO: Rebooting server"
-  /sbin/shutdown -r +2 "auter: System reboot to apply updates" &
+  if [[ ${NOWALLMSG} -eq 1 ]] && shutdown --help | grep -q "no-wall"; then
+    SHUTDOWNOUTPUT=$(/sbin/shutdown --no-wall -r +2 "auter: System reboot to apply updates" &)
```

Comment:
I don't think this is working as intended:
* It's not possible to background the shutdown when it's run in `$()` - the prompt is not returned when running in interactive mode.
* stderr is not captured by the `SHUTDOWNOUTPUT=$()`, it is being printed. You would need `2>&1` at the end of the shutdown command to capture it.

It might simpler to just throw away the output from the shutdown and log that there has been a generic shutdown error that requires further investigation.